pkg/master/controller.go

```diff
@@ -177,7 +178,7 @@ func (c *Controller) UpdateKubernetesService(reconcile bool) error {
 			return err
 		}
 		endpointPorts := createEndpointPortSpec(c.PublicServicePort, "https", c.ExtraEndpointPorts)
-		if err := c.EndpointReconciler.ReconcileEndpoints("kubernetes", c.PublicIP, endpointPorts, reconcile); err != nil {
+		if err := c.EndpointReconciler.ReconcileEndpoints("apiservers", "kubernetes", c.PublicIP, endpointPorts, reconcile, time.Now()); err != nil {
```

Comment:
Shouldnt this (configmap name) be configurable as per https://github.com/kubernetes/kubernetes/pull/34005#discussion_r82660381?